.travis.yml

```diff
@@ -113,7 +113,7 @@ script:
               -DUSE_COTIRE=ON
               -DCOTIRE_MAXIMUM_NUMBER_OF_UNITY_INCLUDES="-j4"
               . &&
-        make install -j4;
+        make install VERBOSE=1 -j4;
```

Comment:
probably c++11 : default "standard" is usually libc++ instead of libstc++

Reply:
yeah, that's what I wanted to know.
But I dont know why it complains if the default is c++11.